README.md

```diff
@@ -135,6 +145,20 @@ resource "nuodbaas_database" "db" {
 }
 ```
 
+### Connecting to the database
+
+The example configuration above also exposes an [`output`](https://developer.hashicorp.com/terraform/language/values/outputs) named `nuosql_args`, which makes available the arguments to supply to the `nuosql` tool in order to establish a client connection to the database.
+The `nuosql` tool can be found in the [NuoDB Client Package](https://github.com/nuodb/nuodb-client).
```

Comment:
It requires [nuodb-client](https://github.com/nuodb/nuodb-client/releases) package v20230228 or later.